examples/ipex/text-generation/ipex_text-generation.ipynb

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
looks great, could you mode it to https://github.com/huggingface/optimum-intel/tree/main/notebooks ?

Reply:
Done